openapiart/bundler.py

```diff
@@ -323,7 +323,16 @@ def _process_yaml_object(self, base_dir, yobject):
                 self._validate_names("^[+a-zA-Z0-9_]+$", "responses", value)
                 self._check_nested_components(value)
         self._resolve_refs(base_dir, yobject)
-
+    
+    def _check_upper_case(self, value):
+        upper_case = False
+                
+        for c in value:
+            if c.isupper():
+                upper_case = True
+        
+        return upper_case
```

Comment:
would suggest breaking early in the function; probably storing False/True in upper_case var is not required

Reply:
Can't see the change here

Reply:
done